.gitignore

```diff
@@ -11,4 +11,5 @@ test/acs-engine-test/acs-engine-test
 
 test/acs-engine-test/acs-engine-test.exe
 pkg/operations/junit.xml
-pkg/operations/kubernetesupgrade/junit.xml
\ No newline at end of file
+pkg/operations/kubernetesupgrade/junit.xml
+pkg/acsengine/templates.go
```

Comment:
If templates.go is not checked in, do we know if this breaks downstream dependencies that rely on this file to be there?  @JackQuincy